.travis.yml

```diff
@@ -0,0 +1,18 @@
+sudo: false
+language: php
+
+php:
+  - 7.2
+  - 7.3
+
+addons:
+  apt:
+    packages:
+      - libmcrypt-dev
+    update: true
+
+before_script:
+  - phpize && ./configure && make
+
+script:
+  - make test REPORT_EXIT_STATUS=1 NO_INTERACTION=1 TESTS="--show-all" | tee tests-output.txt && if grep -q 'TEST SUMMARY$' tests-output.txt ; then exit 1 ; fi
```

Comment:
What's the reason for the tests-output.txt and grep here? Should just the make test run be sufficient? It seems to pass on Travis (which I just enabled).

Reply:
There was no need to grep tests-output.txt.
I fixed it so as not to grep and committed it.